### ISSUE TITLE
Question: How to export symbols for Frameworks developed as private pod / Instruments not displaying symbols for private pod framework

### ISSUE THREAD

**Issue by dessmith** [CocoaPods/CocoaPods#6572]
Hi,

**tldr**; I'm trying to profile my application using Time Profiller however the symbols for my internally developed framework (added to the app as a private Cocoapods) are missing, what can I do to resolve this? _TMUIKit is the name of our framework in the screenshot below._

<img width="824" alt="screen shot 2017-03-21 at 09 35 12" src="https://cloud.githubusercontent.com/assets/5708685/24141000/d75d68c0-0e19-11e7-9ccd-f97d4f0ca9eb.png">

I've looked over a few forums, and all I can see is to set Debug Information to "DWARF with dSYM file" which I have set on both the main project, and the framework.

Is there a flag I can set, somewhere, even if it' overwriting something in the Cocoapods.sh files that would allow the generation of dSYM in for my embedded framework.

Thanks.

Using CocoaPods 1.2.0
Xcode 8.2.1
Swift 2.3

**Comment by dessmith** [comment]
Ok this was my / someone on my team being silly, and set the Profile schema build configuration to be the Debug configuration instead of the Release Configuration.

Hope this helps someone in the same situation.
